js/chat/chat.scss

```diff
@@ -66,12 +66,30 @@ shiny-chat-message {
   .message-icon {
     border-radius: 50%;
     border: var(--shiny-chat-border);
+    height: 2rem;
+    width: 2rem;
+    display: grid;
+    place-items: center;
+    overflow: clip;
+
     > * {
-      margin: 0.5rem;
-      height: 20px;
-      width: 20px;
+      // images and avatars are full-bleed
+      height: 100%;
+      width: 100%;
+      margin: 0 !important;
+      object-fit: contain;
+    }
+
+    > svg,
+    > .icon,
+    > .fa,
+    > .bi {
+      // icons and svgs need some padding within the circle
+      max-height: 66%;
+      max-width: 66%;
```

Comment:
This section differentiates between icons and avatar images so that you get nice rounded full-bleed avatars when you're using an image.

| Before | After |
|--------|--------|
| ![image](https://github.com/user-attachments/assets/33771788-99d6-48e0-89ab-1a69735b5dbc) | ![image](https://github.com/user-attachments/assets/1bf353b9-754e-42b5-b9d4-80690eee1e73) |

Reply:
...but the icons still look great

![image](https://github.com/user-attachments/assets/52ff4ca2-52de-4e61-88e7-f9a2f57026c1)